tests/testthat/test-mle-finder.R

```diff
@@ -0,0 +1,12 @@
+test_that("linalg and optim least-sq coincide", {
+  n_obs <- 32; n_pred <- 4
+  data <- simulate_data(n_obs, n_pred, model = 'linear', seed = 1918)
+  design <- data$design; outcome <- data$outcome
+  via_linalg_out <- hiper_glm(design, outcome, model = 'linear')
+  via_bfgs_out <- hiper_glm(
+    design, outcome, model = 'linear', option = list(mle_solver = 'BFGS')
```

Comment:
Don't mix single and double quotes. Tidyverse style guide recommends double quotes. Same elsewhere in code.